src/Courses/TopicGrades/GradingPage.tsx

```diff
@@ -273,64 +364,131 @@ export const GradingPage: React.FC<GradingPageProps> = () => {
-                            </p>)
-                    }
-                    {selected.user && 
-                      ((currentUserRole !== UserRole.STUDENT && (info?.workbook || _.isNull(selected.problem))) || 
-                      ((!selected.problem && topicFeedback) || (selected.problem && info?.workbook?.feedback))) && 
-                      <Grid container item>
-                          <ListSubheader disableSticky>
-                              <h2>Feedback</h2>
-                          </ListSubheader>
-                          {(currentUserRole === UserRole.STUDENT ? 
-                              <QuillReadonlyDisplay content={selected.problem ? info?.workbook?.feedback : topicFeedback} /> : 
-                              <GradeFeedback 
-                                  workbookId={selected.problem ? info?.workbook?.id : undefined} 
-                                  setGradeAlert={setGradeAlert} 
-                                  defaultValue={selected.problem ? info?.workbook?.feedback : topicFeedback} 
-                                  topicId={topic.id}    
-                                  userId={selected.user.id}
-                              />)}
-                      </Grid>}
-                    <Grid container item alignItems='stretch'>
-                        {selected.problem && selected.user && selected.grade &&
-                        // (selected.problemState?.workbookId || selected.problemState?.studentTopicAssessmentInfoId || selected.problemState?.previewPath) &&
-                            <ProblemIframe
-                                problem={selected.problem}
-                                userId={selected.user.id}
-                                readonly={true}
-                                workbookId={selected.problemState?.workbookId}
-                                studentTopicAssessmentInfoId={selected.problemState?.studentTopicAssessmentInfoId}
-                                previewPath={selected.problemState?.previewPath}
-                                previewSeed={selected.problemState?.previewSeed}
-                            />
-                        }
-                    </Grid>
-                    {(selected.grade || selected.gradeInstance) &&
-                        <Grid container item md={12}>
-                            <AttachmentsPreview
-                                gradeId={selected.grade?.id}
-                                gradeInstanceId={selected.gradeInstance?.id}
-                                // Workbooks don't seem to be loading in the database right now,
-                                // but a professor shouldn't really care about this level. Attachments should show the same for
-                                // all attempts, maybe even all versions?
-                                // workbookId={selected.workbook?.id}
-                            />
-                        </Grid>
-                    }
+                <Grid
```

Comment:
This should be easy enough to hide in another component.

Reply:
I do agree but since this was a "quick fix" I'm going to leave it

in my opinion, this component should have already been broken out into multiple components (like left and right, and sections for the right pane like feedback)